src/config/mod.rs

```diff
@@ -10,9 +10,7 @@ use crate::config::config_type::ConfigType;
 #[allow(unreachable_pub)]
 pub use crate::config::file_lines::{FileLines, FileName, Range};
 #[allow(unreachable_pub)]
-pub use crate::config::lists::*;
-#[allow(unreachable_pub)]
-pub use crate::config::macro_names::{MacroSelector, MacroSelectors};
+pub use crate::config::macro_names::MacroSelector;
```

Comment:
The unused import lint was triggering for `pub use crate::config::lists::*;` and `MacroSelectors` so I removed the import for `crate::config::lists::*;`, and I moved the `MacroSelectors` import down to the test module.